folio-service-tools-test/src/main/java/org/folio/test/util/TestSetUpHelper.java

```diff
@@ -78,7 +74,6 @@ public static void startVertxAndPostgres(Map<String, String> configProperties) {
   public static void stopVertxAndPostgres() {
     CompletableFuture<Void> future = new CompletableFuture<>();
     vertx.close(res -> {
-      PostgresClient.stopEmbeddedPostgres();
```

Comment:
Do we need to stop container here?
Like this: `PostgresClient.stopPostgresTester();`

Reply:
According to Adam Dickmeiss: "Done automatically on process shutdown."